models/pull.js

```diff
@@ -28,6 +28,19 @@ function Pull(data, signatures, comments, commitStatus, labels) {
       this.data.closes = data.closes;
       this.data.connects = data.connects;
    }
+
+   var isPassing = function() {
+      var repoConfig = config.repos.find(repo => repo.full_name === this.data.repo);
+      var requiredBuilds = repoConfig.required_passing_builds;
+
+      var passingBuilds = this.commitStatuses.filter(
+         build => build.state === 'success')
+         .map(build => {return build.data.context;});
```

Comment:
This can just be `.map(build => build.data.context)`

Reply:
right-o. When you're still not comfortable with those ES6 features :cry: